plugin/src/helpers/config.ts

```diff
@@ -24,13 +37,20 @@ export const getNextConfig = async function getNextConfig({
   failBuild = defaultFailBuild,
 }): Promise<NextConfig> {
   try {
-    const { config, appDir, ignore }: RequiredServerFiles = await readJSON(join(publish, 'required-server-files.json'))
+    const { config, appDir, ignore, files }: RequiredServerFiles = await readJSON(
+      join(publish, 'required-server-files.json'),
+    )
     if (!config) {
       // eslint-disable-next-line @typescript-eslint/ban-ts-comment
       // @ts-ignore
       return failBuild('Error loading your Next config')
     }
-    return { ...config, appDir, ignore }
+
+    const routeManifestFile = files.find((f) => f.endsWith(ROUTES_MANIFEST_FILE))
+    const routesManifest = (routeManifestFile ? await readJSON(routeManifestFile) : {}) as RoutesManifest
```

Comment:
Are you sure this is needed? I don't think the routes manifest can be anywhere except `join(publish, 'routes-manifest.json')`